client/extensions/wp-job-manager/components/navigation/index.jsx

```diff
@@ -58,10 +60,22 @@ class Navigation extends Component {
 	}
 
 	render() {
+		const { siteSlug, translate } = this.props;
+
 		return (
-			<SectionNav selectedText="Settings">
-				<SectionNavTabs>{ values( Tabs ).map( tab => this.renderTabItem( tab ) ) }</SectionNavTabs>
-			</SectionNav>
+			<div>
+				<HeaderCake
+					backText={ translate( 'Plugin Overview' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **89** times:
`translate( 'Overview' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/wp-job-manager-header-cake)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).